tests/ci/api_run.sh

```diff
@@ -18,6 +18,18 @@ function uploader {
 set +e
 
 docker ps
+# Ensure E2E engine is available on arm64
```

Comment:
It is true that this api test image is not built in the standard release process, so it can only be processed manually at present. Let's create a new issue to improve this part. Theoretically, this test image should also need to have the same Build entry and support two architectures.